app/javascript/article-form/components/Help/EditorFormattingHelp.jsx

```diff
@@ -108,15 +108,21 @@ export const EditorFormattingHelp = ({ openModal }) => (
         </details>
       </li>
       <li>
-        You can use{' '}
+        Embed Liquid tags to add rich content such as Tweets, YouTube videos,
+        etc. Use the complete URL:&nbsp;
+        <code>{'{% embed Liquid-tag-URL %}'}</code>.{' '}
         <a href="#liquid" onClick={() => openModal('liquidShowing')}>
-          Liquid tags
+          Click here
         </a>{' '}
-        to add rich content such as Tweets, YouTube videos, etc.
+        for a list of supported embeds.{' '}
+        <a href="https://github.com/forem/forem/issues/new?assignees=&amp;labels=&amp;template=bug_report.md">
+          Open an issue
+        </a>{' '}
+        if you experience any issues.
```

Comment:
Could we move it to the modal and add `target="_blank"`? 

The reason for moving it is that we want to keep the default editor view as simple as possible with as little distraction as possible. Even though I do wholeheartedly believe in our community, I'm not sure they will bother to report these bugs that often. 

The reason for `target="_blank"` is to avoid accidentally leaving the editor, especially other two links in the sidebar do not take you away.

Reply:
oh yea thats a good point too!

Reply:
I opt for taking it out. 👍🏾